conf/application.conf

```diff
@@ -84,12 +85,17 @@ microservice {
         welsh-translation: false
       }
 
+      feature-toggle {
+        accessibility-statement = false
+      }
+
       amls-frontend {
         protocol = http
         host = localhost
         port = 9222
         appName: "anti-money-laundering"
         url: ${microservice.services.amls-frontend.protocol}"://"${microservice.services.amls-frontend.host}":"${microservice.services.amls-frontend.port}"/"${microservice.services.amls-frontend.appName}
+        url-accessibility-statement: ${microservice.services.amls-frontend.protocol}"://"${microservice.services.amls-frontend.host}":"${microservice.services.amls-frontend.port}"/"${microservice.services.amls-frontend.appName}"/accessibility-statement"
```

Comment:
if you could change above to:
        url-accessibility-statement: ${microservice.services.amls-frontend.url}"/accessibility-statement"
would be nice :)